src/index.ts

```diff
@@ -52,6 +53,17 @@ const testStore: GroupedTests = {
   },
 };
 
+const logTime = (startTime: number, endTime: number, options: Options) => {
```

Comment:
Rename this function to `calculateExecutionTime`

Also don't send it the entire options object just the precision property.

Finally write a test for this function